README.md

```diff
@@ -16,59 +16,59 @@ Note: If you want to use the plugin on macOS, see our [advice](#macos-advice).
+After receiving this event, the plugin is ready for use. You can now, for example, call this Blueprint's [Connect](#connect) function. Once connected, the [On Connected](#on-connected) event will trigger.
 
 Example:  
 ![example](Images/on_initialized.PNG)
 
 ### On Connected
-Successfully connected to conference. Triggered by the [Connect](#connect) function. Provides the ID of the local participant in its argument.
+Triggered when a client is successfully connected to a conference after calling the [Connect](#connect) function. The event provides the ID of the local participant in its argument.
 
 Example:  
 ![example](Images/on_connected.PNG)
 
 ### On Disconnected
-Disconnected from conference. Triggered when disconnected by any means (in particular by the [Disconnect](#disconnect) function).
+Triggered when a client is disconnected from a conference by any means; in particular, by the [Disconnect](#disconnect) function.
 
 Example:  
 ![example](Images/on_disconnected.PNG)
 
 ### On Remote Participants Changed
-Triggered when participants are added to or removed from the conference. Provides the IDs of the remote participants in its argument.
+Triggered when remote participants are added to or removed from a conference. The event provides the IDs of the participants in its argument.
 
 Example:  
 ![example](Images/on_remote_participants_changed.PNG)
 
 ### On Active Speakers Changed
-Triggered when participants start or stop speaking. Provides the IDs of the current speakers in its argument.
+Triggered when participants start or stop speaking. The event provides the IDs of the current speakers in its argument.
 
 Example:  
 ![example](Images/on_active_speakers_changed.PNG)
 
 ### On Audio Levels Changed
-There are new audio levels available. Provides a string-to-float mapping of participant IDs to their audio levels (0.0 is silent, 1.0 is loudest) in its argument. Triggered by the [Get Audio Levels](#get-audio-levels) function.
+Triggered when there are new audio levels available after calling the [Get Audio Levels](#get-audio-levels) function. The event provides a string-to-float mapping of participant IDs to their audio levels in its argument. A value of 0.0 represents silence and a value of 1.0 represents the maximum volume.
 
 Example:  
 ![example](Images/on_audio_levels_changed.PNG)
 
 ## Functions
 ### Set Token
-Initializes or refreshes the client access token. Takes the token as parameter. Initializes the plugin unless already initialized. Triggers [On Initialized](#on-initialized) if initialization was successful.
+Initializes or refreshes the client access token. The function takes the token as a parameter and initializes the plugin unless already initialized. A successful initialization triggers the [On Initialized](#on-initialized) event.
```

Comment:
"A successful initialization" doesn't sound quite right to me. The "a" article is unnecessary here if we want to keep this wording.